code/modules/clothing/masks/mystic.dm

```diff
@@ -52,20 +52,26 @@
 	flick("happiest_flash", src)
 	canremove = 1
 
+var/list/has_been_shade
```

Comment:
Oh almost forgot: Could you re-run `gen_globals.py` to add this to `globalaccesss.dm`?

Just run `py -3 tools/gen_globals.py vgstation13.dme code/__HELPERS/globalaccess.dm` from the repo root, in a terminal.